src/main/java/carbonconfiglib/gui/screen/MultiChoiceScreen.java

```diff
@@ -52,7 +53,24 @@ public MultiChoiceScreen(Consumer<Result> callback, IChatComponent title, IChatC
 	@SuppressWarnings("unchecked")
 	public void initGui() {
 		super.initGui();
-		output = fontRendererObj.listFormattedStringToWidth(message.getFormattedText(), width-50);
+
+		output = new ArrayList<>();
+		String fullText = message.getFormattedText();
+		String[] lines = fullText.split("\\\\n");
+
+		String activeFormatting = "";
+		for (int i = 0; i < lines.length; ++i) {
+			String line = lines[i];
+
+			if (!line.matches("^§[0-9a-frk-orA-FK-OR].*")) {
+				line = activeFormatting + line;
```

Comment:
This shouldn't be applied here. Since the listFormattedStringToWidth would destroy this anyways again.
You can keep track of the flag but other than that this is to early to apply